### PR TITLE
Fix/Fixes Issue #49

### PR DESCRIPTION
**Pull request by jungeunyoon**

### CODE REVIEW

**Comment by jungeunyoon** [comment]
[Twitter Widget Issue #49 - Design and Implementation [3]](https://trello.com/c/yOgIyB5L/5300-twitter-widget-issue-49-design-and-implementation-3)

File: test/integration/api/twitter.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ describe("Twitter - Integration", ()=> {
 
   it("should get tweets for screen name", done => {
     twitter.getUserTweets("demoComponent", "risevision", (error, tweets)=>{
+
       assert(tweets);
-      assert.equal(Object.keys(tweets).length, 25);
```
Comment (jungeunyoon) [inline test/integration/api/twitter.js]: Removed this as we can no longer guarantee that the module will return 25 tweets with the addition of filtering.

File: test/unit/licensing.js [hunk 2]
```diff
@@ -80,7 +80,6 @@ describe("Licensing - Unit", ()=> {
       clients: ["logging", "system-metrics", "licensing"]
     })
     .then(() => {
-      console.log("HEY")
```
Comment (jungeunyoon) [inline test/unit/licensing.js]: Removed random `console.log`.

File: test/unit/api/twitter.js [hunk 3]
```diff
@@ -13,10 +14,18 @@ const clientMock = {
 simple.mock(twitterWrapper, "getClient").returnWith(clientMock);
 const twitter = require("../../../src/api/twitter");
 twitter.init();
+const twitterRewire = rewire("../../../src/api/twitter");
+const filterTweets = twitterRewire.__get__('_filterTweets');
+
+// sample tweets data
+const onlyValidTweets = require("../../../test/data/valid-tweets.json");
```
Comment (jungeunyoon) [inline test/unit/api/twitter.js]: The sample data comes from actual examples I've used to test edge cases.

File: test/unit/api/twitter.js [hunk 4]
```diff
@@ -3,6 +3,7 @@
 const assert = require("assert");
 const config = require("../../../src/config/config");
 const logger = require("../../../src/logger");
+const rewire = require('rewire');
```
Comment (pcsandford) [inline test/unit/api/twitter.js]: @jungeunyoon is there a reason for single quotes here and on line 18? Thanks!

Comment (jungeunyoon) [reply test/unit/api/twitter.js]: @pcsandford No, it's arbitrary since we didn't add a linting rule for it. I'll change it because now it's bothering me, thanks!
